examples/config.yaml

```diff
@@ -0,0 +1,212 @@
+          exporter: otlp/exporterA # references identifier from exporters section
+          # Configure periodic metric reader interval.
+          interval_millis: 5_000
+    # List of views. Each view consists of a selector defining criteria for which instruments are selected, and a view defining the resulting metric.
+    views:
+      # Add a "kitchen sink" view, using all selector fields, and using all configurable aspects of the view.
+      - selector:
+          # Select instruments with this name, including wildcard matching.
+          instrument_name: my-instrument
+          # Select instruments with this type.
+          instrument_type: COUNTER
+          # Select instruments with this meter name.
+          meter_name: my-meter
+          # Select instruments with this meter version.
+          meter_version: 1.0.0
+          # Select instruments with this meter schema url.
+          meter_schema_url: http://example.com
+        view:
+          # Change the metric name.
+          name: new-instrument-name
+          # Change the metric description.
+          description: new-description
+          # Change the aggregation. In this case, use an explicit bucket histogram with bucket boundaries [1.0, 2.0, 5.0].
+          aggregation:
+            name: explicit_bucket_histogram
+            args:
+              bucket_boundaries: [1.0, 2.0, 5.0]
+          # List of attribute keys to retain. Keys included on measurements and not in this list will be ignored.
+          attribute_keys:
+            - foo
+            - bar
+      # Add a simpler view, which configures the drop aggregation for instruments whose name matches "*.server.duration".
+      - selector:
+          instrument_name: "*.server.duration"
+        view:
+          aggregation:
+            name: drop
+  # Logger provider configuration.
+  logger_provider:
+    resource: resource/resourceA # references identifier from resources section
+    # List of log processors, to be added sequentially. Each log processor has a name and args used to configure it.
+    log_record_processors:
+      # Add batch log processor configured to export with the OTLP exporter.
+      - name: batch
+        args:
+          # Batch log processor takes exporter as an arg
+          exporter: otlp/exporterA # references identifier from exporters section
+          # Configure batch log processor batch size and interval settings.
+          max_queue_size: 50
+          scheduled_delay_millis: 1_000
+          export_timeout_millis: 30_000
+          max_export_batch_size: 200
+  # List of context propagators. Each propagator has a name and args used to configure it. None of the propagators here have configurable options so args are not demonstrated.
+  propagators:
+    - name: tracecontext
+    - name: baggage
+# Using examples provided in https://github.com/MrAlias/otel-schema/issues/8
+instrumentation:
+  java:
```

Comment:
Java Metric Insight has a fairly complex complication which ideally should be merged into this config file as well:

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/jmx-metrics/javaagent/README.md

cc @PeterF778, @trask